lib/helpers/json-to-html-data/pretty-status.js

```diff
@@ -0,0 +1,9 @@
+module.exports = function (status) {
+  var mapping = {
+    'permanent collection': 'Permanent collection',
+    'loan: wellcome trust': 'Loan: Wellcome Trust',
+    'loans: historic and loan': 'Loan'
```

Comment:
I think here, what Emily meant in https://github.com/TheScienceMuseum/collectionsonline/issues/200#issuecomment-259397680 was that 'loans: historic' and 'loan' should both be 'Loan'.  

See http://localhost:8000/objects/co62406/receiver-unit-from-the-eniac-computer-1943-1945-computer-component and http://localhost:8000/objects/co30866/ora-wireless-receiving-triode-1921-thermionic-valve for examples